invenio_accounts/ext.py

```diff
@@ -197,6 +210,18 @@ def delay_security_email(msg):
         self.kvsession_extension = KVSessionExtension(
             session_kvstore, app)
 
+        # Warn if inconsistent configuration is detected
+        sec = app.extensions["security"]
+        local_login = app.config.get("ACCOUNTS_LOCAL_LOGIN_ENABLED", True)
+        x_able = sec.registerable or sec.changeable or sec.recoverable
```

Comment:
Only IMO, very very very minor :)

```suggestion
        local_account_is_editable = sec.registerable or sec.changeable or sec.recoverable
```

I would also have isolated all this block in a method for readability...